src/game.py

```diff
@@ -14,13 +16,38 @@ def __init__(self, ui=UserInterface(), message=Message()):
         self.rules = Rules()
         self.validator = Validator()
         self.symbol = SymbolOptions()
+        self.computer_player = ComputerPlayer()
```

Comment:
How come there's a computer player but not a human player?

Reply:
What @Aoxian said above is what I was getting at here. If you have duck types, you could have `self.player_one` and `self.player_two` and some simple logic at game setup to determine what player types should be assigned. Ideally, that would be the only point where at the game level where you need to make a distinction between player types.